package.json

```diff
@@ -88,5 +88,8 @@
   },
   "peerDependencies": {
     "fp-ts": "^2.16.9"
+  },
+  "dependencies": {
+    "p-retry": "^6.2.1"
```

Comment:
suggestion:I think p-retry should be added under `devDependencies` given that it's bundled into the final distributable.